vars/neoDeploy.groovy

```diff
@@ -6,89 +6,83 @@ import com.sap.piper.ConfigurationType
 import com.sap.piper.tools.ToolDescriptor
 
 
-def call(parameters = [:]) {
+import groovy.transform.Field
+
+@Field String STEP_NAME = 'neoDeploy'
+@Field Set GENERAL_CONFIG_KEYS = []
```

Comment:
@marcusholl: what should be loaded from the `general` section? Is `STEP_CONFIG_KEYS ` sufficient here?